src/resolver.rs

```diff
@@ -1760,6 +1760,9 @@ impl<'i> TypeAnnotator<'i> {
                     return;
                 }
                 self.annotate(statement, StatementAnnotation::value(return_type.get_name()));
+            } else {
+                // Assuming this is a VOID function if no annotation is present
+                self.annotate(statement, StatementAnnotation::value("__VOID"));
```

Comment:
Should I introduce a const global variable for `__VOID` here?

Reply:
We already have one for `VOID` in typesystem.rs, I guess adding another one for the internal variant won't hurt.